src/coreclr/debug/createdump/crashinfo.h

```diff
@@ -35,7 +35,7 @@ extern const std::string GetDirectory(const std::string& fileName);
 extern std::string FormatString(const char* format, ...);
 extern std::string FormatGuid(const GUID* guid);
 
-class CrashInfo : public ICLRDataEnumMemoryRegionsCallback,
+class CrashInfo : public ICLRDataEnumMemoryRegionsCallback, public ICLRDataEnumMemoryRegionsLoggingCallback,
```

Comment:
```suggestion
class CrashInfo : public ICLRDataEnumMemoryRegionsCallback, public ICLRDataLoggingCallback,
```

The logging could also occur in scenarios that are not doing EnumMemoryRegions if we wanted to register the callback right? Even if we only register the logging callback during EnumMemoryRegions right now I'd suggest we pick a name that will still make sense if we make broader usage of this logging in the future.